lambda_functions/two_way_sms/two_way_sms_v2.py

```diff
@@ -182,12 +185,13 @@ def notify_incoming_sms_handler(event: dict, context: any):
         if not valid_message_body(inbound_sms):
             logger.error("The record's message body is invalid: %s", inbound_sms)
 
-            # Push the record to the dead letter queue, and return 200 to have the message removed from the feeder queue.
+            # Push the record to the dead letter queue, and return 200 to have the message removed from the feeder queue
             push_to_sqs(record_body, False)
             if len(event["Records"]) > 1:
                 # TODO - Remove this block as part of #1014.  The problem is that the handler returns if any record has
                 # an invalid message.  It's not a problem now because we only expect one record.
```

Comment:
We can remove this TODO now since we are going to close that ticket. Check with Kyle first!